ui/imports/shared/popups/send/controls/GasSelector.qml

```diff
@@ -16,18 +16,19 @@ Item {
 
     property var selectedAsset
     property string currentCurrency
-
     property var bestRoutes
+
+    property var formatFiat: function () {}
```

Comment:
It would be nice to have expected signatures here, like `property var formatFiat: function (balance) {}` to increase readability.